### DISCUSSION TITLE
HELP!! CONNECTION AND SUBSCRIPTION TO A LIST OF TOPICS.

### DISCUSSION THREAD

**Discussion by SebastianJoan** [emqx/emqx#5501] [Q&A]
i made a app with flutter + MQTT
I need to connect and subscribe to a list of topics once my user logs in, but he only connects, it is not possible to subscribe to these topics, how could I do it, any ideas?

![Captura](https://user-images.githubusercontent.com/36310454/129510653-70d66f98-a022-48ad-8506-9f192fd2d1a5.PNG)

![image](https://user-images.githubusercontent.com/36310454/129510699-257bf883-f596-4b02-be86-41304497a20e.png)

**Comment by qzhuyan** [comment]
| it is not possible to subscribe to these topics

If you get some error return from failed subscription, you can check the `Reason` in SUBACK message.
